files/ja/web/css/path/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: path()
+slug: Web/CSS/path
+tags:
+  - CSS
+  - CSS Function
+  - Function
+  - Reference
+browser-compat: css.types.basic-shape.path
```

Comment:
```suggestion
```

according to https://github.com/mdn/translated-content/pull/6925#discussion_r922824200